src/components/Router.js

```diff
@@ -105,88 +105,93 @@ class Router extends Component {
+      ? <BrowserRouter>
+        <ScrollToTop />
+        <Nav admin={user.admin} />
+        <div className='content'>
+          {userNeedsRegister && <RegisterAlert />}
           <Switch>
 
+            {/* COMMON ROUTES */}
+            <Route
+              path='/login-redirect'
+              render={() => <LoginRedirect />} />
+            <Route
+              path='/forbidden'
+              render={() => <Forbidden />} />
+            <Route
+              path='/profile'
+              featureFlag={'REACT_APP_SHOW_MAXVP'}
+              render={() => <MemberProfile />} />
+            <Route
+              path='/new-member'
+              featureFlag={'REACT_APP_SHOW_MAXVP'}
+              render={() => user.id
+                ? <Redirect to='/' /> /* Allow create member only if user is not yet registered in DB */
+                : <NewMember />} />
             <Route
               path='/event/:id/register'
-              component={EventRegister} />
+              render={() => <EventRegister />} />
             <Route
-              path='/login-redirect'
-              component={LoginRedirect} />
+              path='/event/:id'
+              featureFlag={'REACT_APP_SHOW_MAXVP'}
+              render={props => <EventDetails {...props} user={user} registrations={registrations} />} />
             <Route
-              path='/signup'
-              component={Signup} />
+              path='/events'
+              featureFlag={'REACT_APP_SHOW_MAXVP'}
+              render={() => <UserEvents />} />
+
+            {/* ADMIN ROUTES */}
+            <AdminRoute
+              path='/exec/event/new'
+              render={() => <EventNew />} />
+            <AdminRoute
+              path='/exec/event/:id/edit'
+              render={() => <EventEdit />} />
+            <AdminRoute
+              path='/exec/event/:id' // Need to make sure that this comes after 'new' and 'edit'
+              render={props => <EventView {...props} />} />
+            <AdminRoute
+              path='/exec/home'
+              render={() => <AdminHome />} />
+
+            {/* HOME */}
             <Route
+              exact
               path='/'
-              component={Login} />
+              featureFlag={'REACT_APP_SHOW_MAXVP'}
+              render={() => <UserHome user={user} />} />
 
-            <Redirect to='/' />
+            <Redirect to={user.admin ? '/exec/home' : '/'} />
 
           </Switch>
-        </BrowserRouter >
-    ) : <Loading />
+        </div>
+      </BrowserRouter>
+      : <BrowserRouter>
+        <ScrollToTop />
+        <Switch>
+
+          <Route
+            path='/event/:id/register'
+            component={EventRegister} />
+          <Route
+            path='/login-redirect'
+            component={LoginRedirect} />
+          <Route
+            path='/signup'
+            featureFlag={'REACT_APP_SHOW_MAXVP'}
+            component={Signup} />
+          <Route
+            path='/'
```

Comment:
should this be set to `/exec`? or maybe could be in a different PR

Reply:
This is a good point..
In the [refactor](https://github.com/ubc-biztech/bt-web/pull/187) I tried to address this issue by standardizing the namespace "exec" to "admin", but this route was changed to simply `login`.

Also I realize I think I missed something here. One addition that we should probably have is to check the `REACT_APP_SHOW_MAXVP` feature flag and disallow login if the user is not a `@ubcbiztech.com`?

Reply:
oh yes, good catch LOL, should not let anyone sign in unless they are a biztech exec